tokens/src/themes/light/components/ProgressBar.json

```diff
@@ -3,7 +3,9 @@
     "$type": "color",
     "progress-bar": {
       "bg": { "source": "$progress-bg", "$value": "transparent" },
-      "border": { "source": "$progress-bar-border-color", "$value": "{color.gray.500}" },
+      "border": { "source": "$progress-bar-border-color", "$value": "{color.gray.500}" }
+    },
+    "progress": {
       "bar": {
```

Comment:
[curious/clarification] Can you help me understand why there's a token key of `progress-bar` on line 4, and another `progress.bar` defined on lines 8-9? I believe both of these result in `color.progress-bar.*` in the resulting output CSS variables.

Would it potentially make sense to co-locate tokens under `progress-bar` or `progress.bar` to avoid having 2 different keys result in the same structure within the output CSS variables? Or is there a reason why these are split out separately?

Reply:
Most likely, the reason for the confusion is separate `bg` tokens for `progress-bar` and `progress`. Moved the `bg` color separately for `progress`. Nice catch!

```
.progress {
  display: flex;
  height: var(--pgn-size-progress-bar-height-base);
  overflow: hidden;
  line-height: 0;
  background-color: var(--pgn-color-progress-bg);
  box-shadow: var(--pgn-elevation-progress-bar-box-shadow);

  @include font-size(var(--pgn-typography-progress-bar-font-size));
  @include border-radius(var(--pgn-size-progress-bar-border-radius));
}

.progress-bar {
  display: flex;
  flex-direction: column;
  justify-content: center;
  overflow: hidden;
  color: var(--pgn-color-progress-bar-base);
  text-align: center;
  white-space: nowrap;
  background-color: var(--pgn-color-progress-bar-bg-base);

  @include transition(var(--pgn-transition-progress-bar-transition));
}
```